cmd/tdex/depositmarket.go

```diff
@@ -31,29 +58,534 @@ var depositmarket = cli.Command{
 	Action: depositMarketAction,
 }
 
+type AssetValuePair struct {
+	BaseValue  uint64
+	BaseAsset  string
+	QuoteValue uint64
+	QuoteAsset string
+}
+
+var fragmentationMapConfig = map[int]int{
+	1: 30,
+	2: 15,
+	3: 10,
+	5: 2,
+}
+
 func depositMarketAction(ctx *cli.Context) error {
 	client, cleanup, err := getOperatorClient(ctx)
 	if err != nil {
 		return err
 	}
 	defer cleanup()
 
-	numOfAddresses := ctx.Int64("num_of_addresses")
-	resp, err := client.DepositMarket(
-		context.Background(),
-		&pboperator.DepositMarketRequest{
-			Market: &pbtypes.Market{
-				BaseAsset:  ctx.String("base_asset"),
-				QuoteAsset: ctx.String("quote_asset"),
+	net, err := getNetworkFromState()
+	if err != nil {
+		return err
+	}
+	baseAssetKey := net.AssetID
```

Comment:
we are assuming base asset is always bitcoin, wich is not true.

Better be more explicit here and let the user full specify base & quote asset